lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1313,12 +1308,39 @@ getNetworkInformation (_block0, bp, st) nl = do
       where bomb = error "reached final epoch of the Blockchain!?"
 
 getNetworkParameters
-    :: forall t. ()
-    => (Block, BlockchainParameters, SyncTolerance)
-    -> NetworkLayer IO t Block
+    :: (Block, BlockchainParameters, SyncTolerance)
     -> ApiEpochNumber
     -> Handler ApiNetworkParameters
-getNetworkParameters _ _ _ = throwError err501
+getNetworkParameters (_block0, bp, _st) apiEpochNum = do
+    case apiEpochNum of
+        ApiEpochNumberInvalid txt ->
+            liftHandler $ throwE $ ErrGetNetworkParametersInvalidValue txt
+        ApiEpochNumber epochNum -> do
+            now <- liftIO getCurrentTime
+            let ntrkTip =
+                    fromMaybe slotMinBound (slotAt (W.slotParams bp) now)
+            let currentEpochNum = ntrkTip ^. #epochNumber
+            when (currentEpochNum < epochNum) $ liftHandler $ throwE $
+                ErrGetNetworkParametersNoSuchEpochNo currentEpochNum epochNum
+            pure resp
+        ApiEpochNumberLatest ->
+            pure resp
+  where
+    (W.SlotLength slotLength) = bp ^. #getSlotLength
+    resp :: ApiNetworkParameters
+    resp = ApiNetworkParameters
+        (ApiT $ bp ^. #getGenesisBlockHash)
+        (ApiT $ bp ^. #getGenesisBlockDate)
+        (Quantity slotLength)
+        (Quantity $ W.unEpochLength $ bp ^. #getEpochLength)
+        (bp ^. #getEpochStability)
+        (Quantity $ W.unActiveSlotCoefficient
+            $ bp ^. #getActiveSlotCoefficient )
```

Comment:
We need to multiply with 100 here!

On our integration setup, the coefficient is 100%, but I'm getting

```json
http http://localhost:8090/v2/network/parameters/latest
HTTP/1.1 200 OK
Content-Type: application/json;charset=utf-8
Date: Tue, 28 Jan 2020 10:41:18 GMT
Server: Warp/3.2.27
Transfer-Encoding: chunked

{
    "active_slot_coefficient": {
        "quantity": 1,
        "unit": "percent"
    },
    "blockchain_start_time": "2019-04-25T14:20:57Z",
    "epoch_length": {
        "quantity": 10,
        "unit": "slot"
    },
    "epoch_stability": {
        "quantity": 5,
        "unit": "block"
    },
    "genesis_block_hash": "f8c0622ea4b768421fea136a6e5a4e3b4c328fc5f16fad75817e40c8a2a56a56",
    "slot_length": {
        "quantity": 2,
        "unit": "second"
    }
}
```

Reply:
It would be neat if we could add expectations for all the values in the successful integration test

Reply:
ok, 1 is 100%. you are right

Reply:
will add also additional test in integration test suite

Reply:
added in https://github.com/input-output-hk/cardano-wallet/pull/1306/commits/bafd7cce8f4a647c7883b9e3716c275b9f7b6287